common/src/java/org/apache/hadoop/hive/conf/HiveConf.java

```diff
@@ -2301,7 +2301,7 @@ private static void populateLlapDaemonVarsSet(Set<String> llapDaemonVarsSetLocal
     HIVE_SERVER2_GLOBAL_INIT_FILE_LOCATION("hive.server2.global.init.file.location", "${env:HIVE_CONF_DIR}",
         "Either the location of a HS2 global init file or a directory containing a .hiverc file. If the \n" +
         "property is set, the value must be a valid path to an init file or directory where the init file is located."),
-    HIVE_SERVER2_TRANSPORT_MODE("hive.server2.transport.mode", "binary", new StringSet("binary", "http"),
+    HIVE_SERVER2_TRANSPORT_MODE("hive.server2.transport.mode", "binary", new StringSet("binary", "http", "all"),
```

Comment:
how about using the enumeration you created here for ("binary", "http", "all")?

Reply:
I refactored the previous enum class to be part of common package which this class is part of and use this in other places in the service package.